dlc-manager/src/manager.rs

```diff
@@ -924,6 +926,31 @@ where
         Ok(msg)
     }
 
+    /// Reject a channel that was offered. Returns the [`dlc_messages::channel::Reject`]
+    /// message to be sent as well as the public key of the offering node.
+    pub fn reject_channel(&self, channel_id: &ChannelId) -> Result<(Reject, PublicKey), Error> {
+        let offered_channel = get_channel_in_state!(self, channel_id, Offered, None as Option<PublicKey>)?;
+        let offered_contract = get_contract_in_state!(self, &offered_channel.offered_contract_id, Offered, None as Option<PublicKey>)?;
+
+        if offered_channel.is_offer_party {
```

Comment:
So the offer party can also cancel the offer by calling reject? I hadn't realized this, if that's intended it might be good to reflect this in the method name and documentation.

Reply:
Nope, that was not intentional 🙈, although probably a good feature to add as well later.

I fixed it in 645b9bfb95b722de06251d3077a1c1b997eaa7f6